demisto_sdk/commands/validate/validators/WZ_validators/WZ104_is_wrong_link_in_wizard.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.handlers import DEFAULT_JSON5_HANDLER as json
+from demisto_sdk.commands.content_graph.objects.wizard import Wizard
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Wizard
+
+
+class IsWrongLinkInWizardValidator(BaseValidator[ContentTypes]):
+    error_code = "WZ104"
+    description = "Checks if all fetch integrations have playbook."
+    rationale = "Ensuring all integrations linked to playbook."
+    error_message = "The Following integrations has not linked playbook: {0}"
+    related_field = "fetching_integrations"
+    is_auto_fixable = False
+
+    def integrations_without_playbook(self, content_item: ContentTypes) -> List[str]:
+        content_item_json = json.loads(content_item.text)
+        wizard_json_object = content_item_json.get("wizard", {})
+        set_playbooks = wizard_json_object.get("set_playbook", [])
+        playbooks_link_to_integration = {
+            playbook.get("name", ""): playbook.get("link_to_integration")
+            for playbook in set_playbooks
+        }
+        integrations = []
+        for integration in wizard_json_object.get("fetching_integrations", []):
+            integrations.append(integration.get("name"))
+
+        integrations_without_playbook = []
+        for link in playbooks_link_to_integration.values():
+            if not link:  # handle case that a playbook was mapped to all integration
+                break
+            if link not in integrations:
+                integrations_without_playbook.append(link)
+            else:
+                integrations.remove(link)
```

Comment:
makes sense to cover WZ105 here with a simple change. let's discuss with Yuval what is the right approach.